src/main/play-doc/api/BundleAPI.md

```diff
@@ -81,7 +81,7 @@ Content-Type: application/json
 ]
 ```
 
-Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and port of the service which is currently running.
+Where `10.11.23.22:7001` and `10.11.23.22:7002` are the host and host port of the service which is currently running.
```

Comment:
The host port is mapped to the bind port. External services always use the host port. They are generally one of the same, unless you're using a container, in which case the bind port can be whatever you want it to be.